geometry_msgs/Point p0
geometry_msgs/Point p1
---
bool throughObstacle